MicrosoftFluentUI.podspec

```diff
@@ -82,7 +82,7 @@ Pod::Spec.new do |s|
     core_ios.source_files = ["ios/FluentUI/Configuration/**/*.{swift,h}",
                              "ios/FluentUI/Core/**/*.{swift,h}",
                              "ios/FluentUI/Extensions/**/*.{swift,h}",
-                             "ios/FluentUI/Vnext/Core/FluentUIStyle.generated.swift"]
+                             "ios/FluentUI/Vnext/Core/*.{swift,h}"]
```

Comment:
Is this file up to date with the resource filtering script change? Seems like you have to rebase to validate it?

Reply:
I have rebase this and have clean `pod lib lint`